internal/memdb/memdb.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/apache/arrow/go/v12/arrow/array"
 	"github.com/cloudquery/plugin-sdk/v2/plugins/destination"
 	"github.com/cloudquery/plugin-sdk/v2/schema"
+	"github.com/cloudquery/plugin-sdk/v2/schemav2"
```

Comment:
it's already v2, but we wanted to make another plugin-sdk bump for the source side change.
I'd go with
```suggestion
	"github.com/cloudquery/plugin-sdk/v3/schema"
```

Reply:
no need to do that yet. it's on the destination side and only we have destinations so we can keep it as `v2`

Reply:
then `github.com/cloudquery/plugin-sdk/v2/schema/arrow`?

Reply:
I think `schemav2` is fine as it's a second version of the schema and we want to have an option to access an old one. we can improve this in the future with better versioning but the codebase is not there yet.